views/js/qtiCreator/widgets/static/helpers/widget.js

```diff
@@ -40,13 +40,6 @@ define([
                 'contenteditable' : 'false'
             });
             widget.$container = widget.$original.wrap($wrap).parent();
-            if(widget.$container.length){
-                textNode = widget.$container[0].nextSibling;
-                if(textNode){
-                    //@todo : make text cursor positioning after an inline widget easier
-                    textNode.nodeValue = ' '+textNode.nodeValue;
-                }
-            }
         },
```

Comment:
may cause an issue selecting, setting focus after an inline element. Was issue are you trying to solve by removing this ?

Reply:
Issue that we have an extra space inserted after the tooltip. This is also true for Math elements and other inline elements (some clients are complaining about this). But yes let's discuss this ;)